src/hooks/useMediaQuery.ts

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+
+// Source: https://github.com/juliencrn/usehooks-ts/blob/master/packages/usehooks-ts/src/useMediaQuery/useMediaQuery.ts
+export function useMediaQuery(query: string): boolean {
```

Comment:
**suggestion**: can't we just install this lib? It's not large and hooks are pretty useful